Source/SpacePort/Deprecated/SpaceShipComponents/EngineComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace SpacePort
+{
+    public class EngineComponent : IEngineComponent
+    {
+       
+        public int EngineID { get; set; }
+        public bool IsRunning { get; set; }
+
+        public EngineComponent(bool isRunning)
+        {
+            this.IsRunning = isRunning;
+        }
+
+        public bool GetIsRunning()
+        {
+            return IsRunning;
+        }
+
+        public void Start()
+        {
+            IsRunning = true;
+        }
+
+        public void Stop()
+        {
+            IsRunning = false;
+        }
+    }
+}
```

Comment:
behövs denna class?

Reply:
Antar att ni tänkte använda denna klassen, men i slutändan valde att inte ta med den, eftersom den står under Deprecated :)

Reply:
Seems to be be part of a unit test... maybe...

Reply:
När vi upptäckte att vi fick problem med att strukturera upp vår databas blev vi tvungna att göra en ganska sen overhaul av vårt projekt. Många klasser och moduler blev därför förkastade. Tanken var att vi skulle återimplementera funktionaliteten men vi han inte med detta. Tyvärr så var alla de enhetstester vi gjort baserade på den tidigare strukturen och de blev därmed obsoleta när vi ändrade. 

Lesson learned: Starta med att skapa databasmodellerna innan vi fokuserar på funktionalitet!